test/tests/parser/vector_multimooseenum/tests

```diff
@@ -0,0 +1,25 @@
+[Tests]
+  issues = '#28487'
+  design = 'InputParameters.md'
+
+  [vmme]
+    type = 'RunApp'
+    input = 'vmme.i'
+    requirement = 'The system shall support vectors of MultiMooseEnums as input parameters.'
+    expect_out = 'mme: b; c a; a b c d e; e;'
+  []
+  [vmme_empty]
+    type = RunException
+    input = 'vmme.i'
+    cli_args = "Materials/vmme/mme='b;; a b c d e; e'"
+    requirement = 'The system shall return a usage message when the vector of MultiMooseEnums includes an empty entry.'
+    expect_err = "missing required parameter 'Materials/vmme/mme'"
```

Comment:
I must have commented on the commit, I dont see my comment anymore

this isnt the right message, it should say something explicit like "one entry in the vector is empty and this is not allowed"
let's create a feature request issue if you're running out of time on this

Reply:
I gave it another try.  Think this now does what you requested.

Reply:
awesome. thanks